src/UAEventEmitter.ts

```diff
@@ -27,12 +28,24 @@ export class UAEventEmitter {
         }
       );
     }
-    
-    InteractionManager.runAfterInteractions(() => {
+
+    if (Platform.OS === 'ios' && AppState.currentState !== 'active') {
```

Comment:
This breaks background push, we need to be able to process events even if the app is not active